src/string/CreditCard.ts

```diff
@@ -0,0 +1,168 @@
+/**
+ * TODO: Add module comment
```

Comment:
🔧 May want to add a note here. I think an abbreviated version of the PR description would be good.

Reply:
Oops, completely overlooked these generated comments